### ISSUE TITLE
Modules: New 'Modules and Pipettes' Sidebar Nav and Page

### ISSUE THREAD

**Issue by umbhau** [Opentrons/opentrons#1734]
As a Run App user, I would like to have a dedicated page for my modules and pipettes, so that they're not all crowded on the same page as my robot settings.

## Acceptance Criteria
- [ ] Robot card in robot sidebar has sub-nav for 'Robot' and 'Modules and Pipettes'
- [ ] New 'Modules and Pipettes' page 
- [ ] 'Pipettes' card moved from existing 'Robot' page to new 'Modules and Pipettes' page

**Comment by pantslakz** [comment]
![screen shot 2018-06-23 at 3 23 10 am](https://user-images.githubusercontent.com/2686086/41804310-8c369b5c-7694-11e8-833a-80733d2df0fa.png)

@mcous @howisthisnamenottakenyet @umbhau 
Heyas, some notes re this design:
- side nav panel menu hierarchy is still being worked through - but one option could be to remove the toggle entirely - since it's unclear what it toggles currently, and instead rely on the connect button in robot settings page. 
- removed connection success banner for now. if the connection is successful - it populates in the card. Let's start with that and add more visual cues once we understand the states of modules better.
-

**Comment by Kadee80** [comment]
Moving getting and displaying module info into its own ticket. This ticket will be implemented with this screen to avoid being blocked by API endpoint work.

<img width="825" alt="screen shot 2018-06-26 at 12 15 07 pm" src="https://user-images.githubusercontent.com/3430313/41925450-9bb32bb0-793a-11e8-8d25-908b52e9a407.png">
